Bottom: 7709d0c44f18a52b4a6c6062c17fbcb244f8f545
Top:    0aee89d276725a4aa041b9e2e48f7fee0557894e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2016-02-18 17:11:14 +0000

Change scroll-wheel bindings to match more GTK applications










---

diff --git a/src/gpcb-menu.res.in b/src/gpcb-menu.res.in
index 4cea179..bc73859 100644
--- a/src/gpcb-menu.res.in
+++ b/src/gpcb-menu.res.in
@@ -25,18 +25,18 @@ Mouse =
     up-shift-ctrl = { Mode(Notify) Mode(Restore) Display(ToggleRubberbandMode) }
   }
   Up = {
-    Zoom(0.8)
-    shift = Scroll(up)
-    ctrl = Scroll(left)
+    Scroll(up)
+    shift = Scroll(left)
+    ctrl = Zoom(0.8)
   }
   Down = {
-    Zoom(1.25)
-    shift = Scroll(down)
-    ctrl = Scroll(right)
+    Scroll(down)
+    shift = Scroll(right)
+    ctrl = Zoom(1.25)
   }
 # If you want zoom to center, do this instead.
-  #Up = { {Zoom(0.8) Center()} }
-  #Down = { {Zoom(1.25) Center()} }
+  #Up = { ctrl = {Zoom(0.8) Center()} }
+  #Down = { ctrl = {Zoom(1.25) Center()} }
 
   scroll-left = {
     Scroll(left)
